content/en/docs/installation/upgrading/upgrading-0.16-1.0.md

```diff
@@ -0,0 +1,124 @@
+
+We have released our `cert-manager.io/v1` API that replaces `cert-manager.io/v1alpha2`.
+Since the legacy version for Kubernetes 1.15 and below only supports one CRD version
+you have to transition all resources to `cert-manager.io/v1`.
+
+
+This makes for a fairly significant breaking change for users, as **all**
+cert-manager resources will need to be updated to reflect these changes.
+Ingress annotations will stay the same, this means if you only use ingress-shim 
+you do not have to convert these resources over but it is recommended. 
+However you should convert the (Cluster)Issuers and delete the old CRD versions.
+
+This upgrade should be performed in a few steps:
+
+1. Back up existing cert-manager resources. See the backup section.
+
+2. [Uninstall cert-manager](../../uninstall/).
+
+3. Ensure the old cert-manager CRD resources have also been deleted: `kubectl get crd | grep cert-manager.io`
+
+4. Update the `apiVersion` on all your backed up resources from
+   `cert-manager.io/v1alpha2` to `cert-manager.io/v1`. See the converting section for that.
+
+5. Re-install cert-manager v1.0 from scratch according to the [installation
+   guide](../../).
+
+6. Apply the backed up resources again.
+
+You must be sure to properly **backup**, **uninstall**, **re-install** and
+**restore** your installation in order to ensure the upgrade is successful.
+
+#### Backing up resources
+You can backup the custom resources you or cert-manager created using the following `kubectl` command:
+```bash
+kubectl get -o yaml \
+   --all-namespaces \
+   issuer,clusterissuer,certificates,certificaterequests > cert-manager-backup.yaml
+```
+
+*Note that this will not export private keys or secrets.*
+
+#### Uninstall cert-manager
+Next step is to uninstall cert-manager. 
+This will cause a temporary halt to renewal of certificates but will not affect any TLS traffic.
+
+On how to do this depends on how you installed cert-manager.
+
+Using Helm:
+```bash
+$ helm --namespace cert-manager delete cert-manager
+```
+
+Using `kubectl`:
+```bash
+kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/vX.Y.Z/cert-manager.yaml
+```
+
+Make sure you also have delete the CRDs. This will delete all cert-manager resources also, make sure your backup is complete.
+You can do this manual doing:
+```bash
+kubectl delete crd certificaterequests.cert-manager.io
+kubectl delete crd certificates.cert-manager.io
+kubectl delete crd challenges.acme.cert-manager.io
+kubectl delete crd clusterissuers.cert-manager.io 
+kubectl delete crd issuers.cert-manager.io
+kubectl delete crd orders.acme.cert-manager.io
+```
+
+For more info see the [uninstall cert-manager guide](../../uninstall/).
+
+#### Converting resources
+
+You can use our [kubectl plugin](../../../usage/kubectl-plugin/) to automatically convert you backup from v1alpha2 to v1 using the following command:
```

Comment:
```suggestion
You can use our [kubectl plugin](../../../usage/kubectl-plugin/) to automatically convert your backup from v1alpha2 to v1 using the following command:
```

Perhaps this step should be performed immediately after backing up, so that the user has time to make any manual adjustments to the converted resources.

And perhaps we should recommend that they do a practice (--dry-run) install of the converted resources using a test cluster with cert-manager v1 installed, so that they can iron out any validation failures before uninstalling the existing cert-manager.